env.example.json

```diff
@@ -11,6 +11,7 @@
   "SHAPESHIFT_API_KEY": "xxxx",
   "SHAPESHIFT_CLIENT_ID": "xxxx",
   "SHAPESHIFT_SECRET": "xxxx",
+  "FOX_EXCHANGE_API_KEY": "xxxx",
```

Comment:
This needs to be fixed. Change it to `"x_FOX_INIT": {"apiKey": ""}` and move it to the section below (in alphabetical order).